builtin/credential/cert/path_config.go

```diff
@@ -0,0 +1,63 @@
+package cert
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfig(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "config",
+		Fields: map[string]*framework.FieldSchema{
+			"verify_cert": &framework.FieldSchema{
+				Type:        framework.TypeBool,
+				Default:     true,
+				Description: `If set, during renewal, checks if the presented client certificate matches the client certificate used during login. Defaults to true.`,
+			},
+		},
+
+		Callbacks: map[logical.Operation]framework.OperationFunc{
+			logical.UpdateOperation: b.pathConfigWrite,
+		},
+	}
+}
+
+func (b *backend) pathConfigWrite(
+	req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
+	verifyCert := data.Get("verify_cert").(bool)
+
+	entry, err := logical.StorageEntryJSON("config", config{
+		VerifyCert: verifyCert,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	if err := req.Storage.Put(entry); err != nil {
+		return nil, err
+	}
+	return nil, nil
+}
+
+// Config returns the configuration for this backend.
+func (b *backend) Config(s logical.Storage) (*config, error) {
+	entry, err := s.Get("config")
+	if err != nil {
+		return nil, err
+	}
+	if entry == nil {
+		return nil, nil
```

Comment:
Return a default configuration here instead. That way you don't need to check for the `nil` case elsewhere in the backend. With the change I mentioned above this logic will work properly.